browser/static/js/main_zstack.js

```diff
@@ -752,7 +752,7 @@ const _calculateMaxHeight = () => {
 function upload_file(cb) {
   $.ajax({
     type: 'POST',
-    url: `upload_file/${project_id}`,
+    url: document.location.origin + `/upload_file/${project_id}`,
```

Comment:
```suggestion
    url: `${document.location.origin}/upload_file/${project_id}`,
```

Can you test if this works?

Also, can you update `track.js` to use this as well?